docs/api.adoc

```diff
@@ -392,7 +392,89 @@ Show a smart contract that was hidden before by the command `emerald_hideContrac
 
 === emerald_updateContract
 
-TBD
+Update contract information. Contract address and chain information are used to identify the contract, and may not be updated. 
+
+*Parameters*:
+
+    * `contract` (Object)
+    ** `address` (String) - hex-encoded 20 bytes public address
+    ** `name` (String, optional) - contract name
+    ** `description` (String, optional) - contract description
+    ** `bytecode` (String, optional) - hex-encoded compiled contract
+    ** `abi` (Array) - JSON format for a contract ABI, should be an array of function and/or event descriptions as defined https://github.com/ethereum/wiki/wiki/Ethereum-Contract-ABI[here]. Each operator should have the following properties:
+    *** `name` (String) - the name of the function
+    *** `inputs` (Array) - an array of objects, each of which contains a name and a type
+    *** `outputs` (Array) - an array of objects, each of which contains a name and a type
```

Comment:
ok, makes sense. i wasn't sure if we should allow editing`abi` and `bytecode` in case the user needs to fix a typo... but you're right, they can just reimport.